src/components/utilities/image.js

```diff
@@ -19,5 +19,8 @@ Image.propTypes = {
   className: PropTypes.string,
   imageArr: PropTypes.arrayOf(PropTypes.stings).isRequired
 }
+Image.defaultProps = {
```

Comment:
Before we commit to this `deafultProps` pattern for the `className` prop. Let's check the browser and make sure the `class` does not compile down to `class=""`.

If it does not we are good to go. Just add a line break between `defaultProps` obj and the `propTypes` obj.

If so, lets remove this default props and add `/* eslint-disable react/require-default-props */` above the declaration of the `propTypes` to stop the error.

Reply:
Understood, I just undid those changes for now. We can discuss later about whether we want to require default props or ignore them in the `.eslintrc`.